### PR TITLE
W-11558151-Legacy-vpn

### PR DESCRIPTION
**Pull request by luanamulesoft**
# Writer's Quality Checklist

Before merging your PR, did you:

- [x] Run spell checker
- [x] Run link checker to check for broken xrefs
- [x] Check for orphan files
- [x] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+[TIP]
+Actual configuration steps, and functionality may differ based on the type of device you use. If supported on your device, use Border Gateway Protocol (BGP) routing.
+
+=== Verify Anypoint VPN Entitlements
+
+. Log in to Runtime Manager using an account with the required roles and permissions. See xref:vpn-create-arm.adoc#prerequisites[Create an Anypoint VPN Connection] for more information.
+. Navigate to the Business Group where your VPC resides (if not in the parent organization).
+** If the VPNs option is present in the left-hand menu, you have the necessary entitlements to create an Anypoint VPN connection.
+** When the VPNs option is not is present:
```
Comment (dnishant) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]: *nit* typo -> is not present

**Review by dnishant — DISMISSED** [review]

File: modules/ROOT/pages/legacy-vpn-migration.adoc (hunk 1 above)
Comment (luanamulesoft) [reply modules/ROOT/pages/legacy-vpn-migration.adoc]: fixed

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+If you are using legacy VPN system, you can now migrate from Legacy VPN to Anypoint VPN on a per-route basis, and delete your legacy VPN from the Anypoint Console UI.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
If you are using a legacy VPN system, you can now migrate to Anypoint VPN on a per-route basis, and delete your legacy VPN from the Anypoint Console UI.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+All VPN connections configured prior to November 2018 are eligible for migration to the new Anypoint VPN solution.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
All VPN connections configured before November 2018 are eligible for migration to Anypoint VPN.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+== Prerequisites
+
+=== Check Device Requirements
+
+Before migrating to Anypoint VPN, ensure that your device meets all necessary requirements. See xref:vpn-about.adoc[Anypoint VPN] for more information.
+
+[TIP]
+Actual configuration steps, and functionality may differ based on the type of device you use. If supported on your device, use Border Gateway Protocol (BGP) routing.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
Actual configuration steps and functionality may differ based on the type of device you use. If your device supports it, use Border Gateway Protocol (BGP) routing.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+=== Verify Anypoint VPN Entitlements
+
+. Log in to Runtime Manager using an account with the required roles and permissions. See xref:vpn-create-arm.adoc#prerequisites[Create an Anypoint VPN Connection] for more information.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
. Using an account with the required roles and permissions, log in to Runtime Manager. See xref:vpn-create-arm.adoc#prerequisites[Create an Anypoint VPN Connection] for more information.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+. Navigate to the Business Group where your VPC resides (if not in the parent organization).
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
. If it's not in the parent organization, navigate to the Business Group where your VPC resides.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+** If the VPNs option is present in the left-hand menu, you have the necessary entitlements to create an Anypoint VPN connection.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
** If the VPNs option is present in the navigation menu, you have the necessary entitlements to create an Anypoint VPN connection.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+** When the VPNs option is not present:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc:28]:
```suggestion
** If the VPNs option is not present:
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc (hunk 7 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/legacy-vpn-migration.adoc]: I'm not sure this should be a bullet. Maybe just a regular sentence under the step

so 

```
If it's not in the parent organization, navigate to the Business Group where your VPC resides.
+
If the VPNs option is present in the left-hand menu, you have the necessary entitlements to create an Anypoint VPN connection.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc (hunk 8 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion

```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+*** If your VPC resides in the parent organization, contact Customer Success Manager for assistance with this request.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
. If your VPC resides in the parent organization, contact Customer Success Manager for assistance with this request.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+*** If your VPC resides in a child Business Group, assign the VPN entitlements:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
. If your VPC resides in a child Business Group, assign the VPN entitlements:
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+**** Log into Access Management.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
.. Log into Access Management.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+**** Go to Organization.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
.. Go to Organization.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+**** Select the Business Group.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
.. Select the Business Group.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+**** Use the sliders in the Business Group Info box to assign the required number of VPNs.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
.. Use the sliders in the Business Group Info box to assign the required number of VPNs.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+
+
+=== Configure VPN Device
+
+Download the VPN configuration file from Runtime Manager, and configure your VPN device. See xref:vpn-about.adoc#ipsec-settings[Anypoint VPN] for more information on IPsec Settings.
+
+Running both the legacy and new VPN using the same encryption domains for a single VPC creates asymmetric routing scenarios, and breaks connectivity to the VPC.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
Running both the legacy and Anypoint VPN using the same encryption domains for a single VPC creates asymmetric routing scenarios and breaks connectivity to the VPC.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+You can validate connectivity via the new VPN using a single host (a /32 encryption domain). Either create a single /32 static route, or propagate this route via BGP.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
You can validate connectivity via Anypoint VPN using a single host (a /32 encryption domain). Either create a single /32 static route, or propagate this route via BGP.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+
+Make sure you follow these rules when routing traffic:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
When routing traffic, follow these guidelines :
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+
+* When your VPC route table has multiple routes, use the most specific route.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
* If your VPC route table has multiple routes, use the most specific route.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+* When your VPC route table has duplicate routes, use the manually configured route pointing to the legacy VPN.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
* If your VPC route table has duplicate routes, use the manually configured route pointing to the legacy VPN.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc (hunk 10 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/legacy-vpn-migration.adoc]: the next couple edits change the structure just a bit so be sure to check the formatting

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+
+[NOTE]
+When the same route is set for the Legacy VPN and Anypoint, the traffic goes to the Legacy VPN.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
When the same route is set for the legacy VPN and Anypoint VPN, the traffic goes to the legacy VPN.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 21]
```diff
@@ -0,0 +1,87 @@
...
+
+See https://help.mulesoft.com/s/article/How-to-View-the-VPC-Route-Table-via-the-UI[How to View the VPC Route Table via the UI] for information on how to check your VPC route table.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
See https://help.mulesoft.com/s/article/How-to-View-the-VPC-Route-Table-via-the-UI[How to View the VPC Route Table via the UI^] for information on how to check your VPC route table.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+
+
+=== Create New Anypoint VPN in Runtime Manager
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
=== Create an Anypoint VPN in Runtime Manager
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+
+Follow the step-by-step instructions in xref:vpn-create-arm.adoc[Create an Anypoint VPN Connection] to create a new Anypoint VPN in Runtime Manager.
+
+Provisioning the new VPN before deleting the legacy one will reduce downtime.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
Provisioning the Anypoint VPN before deleting the legacy one will reduce downtime.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 24]
```diff
@@ -0,0 +1,87 @@
...
+To reduce the impact on connectivity while validating the new VPN connection, establish the new VPN for a /32 encryption domain (a single host address in your network).
+When migrating from the legacy VPN to Anypoint VPN, you can update your new VPN connection to include all the required encryption domains.
+
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc:61]:
```suggestion
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 25]
```diff
@@ -0,0 +1,87 @@
...
+
+== Migrate and Test your Legacy VPN to Anypoint VPN
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
== Migrate the Legacy VPN to Anypoint VPN
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 26]
```diff
@@ -0,0 +1,87 @@
...
+
+Once you have created your new Anypoint VPN:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Legacy VPN* in the left menu.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
. From the navigation menu, select *Legacy VPN*.
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 27]
```diff
@@ -0,0 +1,87 @@
...
+. Select the legacy VPN you want to migrate. +
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
. Select the legacy VPN you want to migrate. 
+
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 28]
```diff
@@ -0,0 +1,87 @@
...
+image:legacy-vpn-select.png[Select Legacy VPN]
+. Disable a route and then test the connection going through the Anypoint VPN.
+.. Repeat this step for each route. +
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
.. Repeat this step for each route. 
+
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 29]
```diff
@@ -0,0 +1,87 @@
...
+image:legacy-vpn-disable-routes.png[Disable Legacy VPN Routes]
+. Once you disable all Legacy VPN routes, select *Delete VPN*. +
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
. Once you disable all legacy VPN routes, select *Delete VPN*. 
+
```

File: modules/ROOT/pages/legacy-vpn-migration.adoc [hunk 30]
```diff
@@ -0,0 +1,87 @@
...
+image:legacy-vpn-delete.png[Delete Legacy VPN]
+. A pop-up message appears prompting you to confirm the deletion. Click *Delete*. +
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/legacy-vpn-migration.adoc]:
```suggestion
. When prompted, confirm the deletion. Click *Delete*. 
+
```

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**